src/main/java/org/opensearch/eval/Constants.java

```diff
@@ -61,9 +89,31 @@ public class Constants {
             "          }";
 
     /**
-     * The metrics index mapping.
+     * THe name of the index that stores the results from each query in a query set run.
      */
-    public static final String METRICS_MAPPING_INDEX_MAPPING = "{\n" +
+    public static final String QUERY_RESULTS_INDEX_NAME = "srw_query_results";
+
+    /**
+     * The query sets index mapping.
+     */
+    public final static String QUERY_RESULTS_MAPPING = "{\n" +
```

Comment:
Mapping for query results.

Reply:
Thinking of it, the number of results might be interesting as well to answer the question "How does search config X affect the recall of my search application compared to search config Y?".